arch/lkl/include/asm/unistd.h

```diff
@@ -24,3 +24,83 @@
 	    __ASCII_MAP(x, __SC_ASCII, __VA_ARGS__) ")\\n\"\n"		\
 	    ".ascii \"#endif\\n\"\n"					\
 	    ".section .text\n");
+
+/*
+ * Unsupported system calls due to lack of support in LKL (e.g. related to
+ * virtual memory, signal, user processes). We also only support 64bit version
+ * of system calls where we have two version to keep the same API across 32 and
+ * 64 bit hosts.
+ */
+
+#define __IGNORE_restart_syscall
+#define __IGNORE_exit
+#define __IGNORE_fork
+#define __IGNORE_execve
+#define __IGNORE_ptrace
+#define __IGNORE_alarm
+#define __IGNORE_pause
+#define __IGNORE_kill
+#define __IGNORE_brk
+#define __IGNORE_uselib
+#define __IGNORE_swapon
+#define __IGNORE_mmap
+#define __IGNORE_munmap
+#define __IGNORE_swapoff
+#define __IGNORE_clone
+#define __IGNORE_mprotect
+#define __IGNORE_init_module
+#define __IGNORE_quotactl
+#define __IGNORE_msync
+#define __IGNORE_mlock
+#define __IGNORE_munlock
+#define __IGNORE_mlockall
+#define __IGNORE_munlockall
+#define __IGNORE_mremap
+#define __IGNORE_rt_sigreturn
+#define __IGNORE_rt_sigaction
+#define __IGNORE_rt_sigprocmask
+#define __IGNORE_rt_sigpending
+#define __IGNORE_rt_sigtimedwait
+#define __IGNORE_rt_sigqueueinfo
+#define __IGNORE_rt_sigsuspend
+#define __IGNORE_sigaltstack
+#define __IGNORE_vfork
+#define __IGNORE_mincore
+#define __IGNORE_madvise
+#define __IGNORE_getdents /* we use the 64 bit counter part instead */
```

Comment:
"counterpart" should be just one word.
(with the space, it's easy to mis-read it as "64 bit counter")